### PR TITLE
Adding Pensu as owners

### PR DESCRIPTION
**Pull request by Pensu**
CC: @mbbroberg

### CODE REVIEW

**Comment by Pensu** [comment]
@mbbroberg I am member of Kubernetes github org. I assumed that covered kubernetes-sigs as well. Do I need to do that separately?

**Review by rajula96reddy — APPROVED** [review]
/lgtm

**Comment by rajula96reddy** [comment]
> @mbbroberg I am member of Kubernetes github org. I assumed that covered kubernetes-sigs as well. Do I need to do that separately? CC: @rajula96reddy

I think we are good. Once we merge this, you will have access.

**Comment by mrbobbytables** [comment]
You will need to be added to the k-sigs org via the k/org repo. As a member of Kubernetes, you're implicitly allowed to join the other orgs, its just not automatic.

File: OWNERS [hunk 1]
```diff
@@ -5,4 +5,5 @@ approvers:
   - kaslin
   - mbbroberg
   - parispittman
-  - rajula96reddy 
+  - rajula96reddy
+  - Pensu
```
Comment (palnabarun) [inline OWNERS:9]: nit: can you please sort the list? :)
```suggestion
  - Pensu
  - rajula96reddy
```

**Review by palnabarun — CHANGES_REQUESTED** [review]

**Comment by mbbroberg** [comment]
@Pensu +1 to what Bob said -- we can get the overall org in the other repo. Post nit this lgtm too.

**Comment by Pensu** [comment]
@palnabarun Sorted! PTAL.

**Pull request merged**